GiniVision/Classes/FilePickerManager.swift

```diff
@@ -12,6 +12,7 @@ import MobileCoreServices
 internal final class FilePickerManager:NSObject {
         
     var didPickFile:((GiniVisionDocument) -> ()) = { _ in }
+    fileprivate var acceptedDocumentTypes = [kUTTypePDF as String] + GiniImageDocument.acceptedImageTypes
```

Comment:
Could move `[kUTTypePDF as String]` to `GiniPDFDocument.acceptedType` as it belongs to that classes responsibility.